src/prog_translator.h

```diff
@@ -33,8 +34,22 @@ class ProgTranslator {
     std::vector<double> col_ubs;
     std::vector<double> col_lbs;
     CoinPackedMatrix m;
+    double inf;
   };
 
+  /// a helper function to translate capacities
+  ///
+  /// @param grp the exchange group
+  /// @param request whether this is a request or not
+  /// @param faux_id a false arc id for surplus supply
+  /// @param cap_rows the capacity rows which will be appended to
+  /// @param ctx the translation context
+  static void XlateCaps(ExchangeNodeGroup* grp,
```

Comment:
I am still not a huge fan of using `X` to mean `Trans`.  (GCSG: http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#General_Naming_Rules).  However, there is wiggle room here since it is more important to be consistent. I dunno. Your call. My 2 drachmas.